src/model/entity/PostalCode.php

```diff
@@ -1,13 +1,19 @@
 <?php
 class PostalCode {
+  private int $postalCodeId;
```

Comment:
How come it was needed? The postalcode itself can be a problem or in unique cases like two venue in one city a duplicate?